docs/quickstart/README.md

```diff
@@ -70,6 +72,7 @@ minikube   Ready    master   79s   v1.18.3
 ```
 
 [Troubleshoot 1: Minikube fails to start with a Docker CLI context error](#troubleshooting)
+[Troubleshoot 4: Right kubectl missing...](#troubleshooting)
```

Comment:
```suggestion
[Troubleshoot 1: Minikube fails to start with a Docker CLI context error](#troubleshooting)

[Troubleshoot 4: Right kubectl missing...](#troubleshooting)
```

Reply:
If you see the render on GitHub, Troubleshoot 4 immediately follows Troubleshoot 1. It's hard to tell that there are 2 troubleshoot links.